RePoE/parser/modules/gems.py

```diff
@@ -236,7 +236,7 @@ def _convert_gepl(self, gepl, gess, gesspl, multipliers, is_support):
             if ge["Id"] in self.granted_effect_quality_stats:
                 for geq in self.granted_effect_quality_stats[ge["Id"]]:
                     for k, v in zip(geq["StatsKeys"], geq["StatsValuesPermille"]):
-                        q_stats.append({"id": k["Id"], "value": v, "set": geq["SetId"]})
+                        q_stats.append({"id": k["Id"], "value": v, "set": geq["SetId"], weight: geq["Weight"]})
```

Comment:
The weight key is missing quotation marks.

```suggestion
                        q_stats.append({"id": k["Id"], "value": v, "set": geq["SetId"], "weight": geq["Weight"]})
```

Reply:
whops, nice find